### PR TITLE
Switch to GH Forms

### PR DESCRIPTION
**Pull request by gilbertsoft**
## The Problem/Issue/Bug:

Since some time GitHub Forms are available which give the users a much better experience by simply filling a form instead of the single abstract textarea.

## How this PR Solves The Problem:

This patch removes the legacy issue templates and adds GitHub Forms instead. Also links to get support to the DDEV Discord and Stack Overflow are added.

## Release/Deployment notes:

<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/3673"></a>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: 🐞 Bug report
+description: Create a report to help us improve.
+labels: [bug]
```
Comment (gilbertsoft) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]: This adds the label bug to each bug report to enable simple filtering for bugs or feature request.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name: 🚀 Feature request
+description: Suggest an idea for this project.
+labels: [enhancement]
```
Comment (gilbertsoft) [inline .github/ISSUE_TEMPLATE/feature_request.yml:3]: This adds the label enhancement to each feature request to enable simple filtering for bugs or feature request.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```
Comment (gilbertsoft) [inline .github/ISSUE_TEMPLATE/config.yml:1]: This forces the user to use one of the templates and does not longer allow blank issues.

Comment (rfay) [reply .github/ISSUE_TEMPLATE/config.yml:1]: I often open blank issues when the issue is simple and known. Better than just keeping notes privately.

**Pull request merged**